broker/tasks/alb.py

```diff
@@ -36,9 +40,11 @@ def get_lowest_used_alb(listener_arns) -> tuple[str, str]:
     return listener_data["LoadBalancerArn"], listener_data["ListenerArn"]
 
 
-def get_lowest_dedicated_alb(service_instance, db):
+def get_potential_listeners(service_instance):
```

Comment:
```suggestion
def get_potential_listeners_for_dedicated_instance(service_instance):
```

I think it reads a bit better if we spell out that this function is for dedicated instances
(that makes me wonder if this should be an instance method, too, but that's probably out of scope)